deploy/olm-catalog/courier/qdr-operatorsource.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorSource
+metadata:
+  name: qdr-operator
+  namespace: openshift-marketplace
+spec:
+  type: appregistry
+  endpoint: https://quay.io/cnr
+  registryNamespace: myeung
```

Comment:
I think if we are committing an operatorsource to this repo it should use the interconnectedcloud organisation. Really though I think we want to prepare to submit it to the upstream operators registry.

Reply:
change the namespace to ''interconnectedcloud"